scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'all-release-branches',
-            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:balrog:server:release': 'all-release-branches',
```

Comment:
I've pushed the refactored patch. Two observations here:
1. In your in-tree [patch](https://reviewboard.mozilla.org/r/113722/diff/2#index_header) shouldn't you have `esr` entry in `BALROG_SERVER_SCOPES` and `BALROG_SCOPE_ALIAS_TO_PROJECT` ?

2. I suppose `aurora` is in the `release` section because of possible staging-beta out of it, but for some reason in my head something is not right with respect to future aurora nightlies once we land this.